HealthcareSystem/Backend/Model/Manager/Room.cs

```diff
@@ -6,11 +6,13 @@
 
 using Model.Enums;
 using System;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace Model.Manager
 {
-   public class Room
-   {
+    public class Room
+    {
+        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Number { get; set; }
```

Comment:
Does this need to be annotated with [Key]? I think Entity Framework only infers keys without the annotation if they're named Id.

Reply:
It is true, i will rename it to be Id